pages/Account.qml

```diff
@@ -349,8 +349,9 @@ Rectangle {
                 id: addNewAccountCheckbox 
                 visible: !selectAndSend
                 border: false
-                checkedIcon: "qrc:///images/plus-in-circle-medium-white.png" 
-                uncheckedIcon: "qrc:///images/plus-in-circle-medium-white.png" 
+                checkedIcon: FontAwesome.minusCircle
```

Comment:
2019-11-25 06:31:38.540	W qrc:/pages/Account.qml:353: ReferenceError: FontAwesome is not defined